README.adoc

```diff
@@ -454,6 +455,48 @@ tomcat:
 <4> Optional parameter to configure sleeping time between log downloading. You can set in seconds using _s_ or miliseconds using _ms_. By default time unit is miliseconds and value 500.
 <5> Optional parameter to configure number of retries to be done. By default 10 iterations are done.
 
+Custom Await strategy:
+
+[source, java]
+.org.arquillian.cube.docker.impl.await.CustomAwaitStrategyImpl
+----
+public class CustomAwaitStrategyImpl implements AwaitStrategy {
+
+  Await params;
+  DockerClientExecutor dockerClientExecutor;
+  Cube<?> cube;
+
+  public void setCube(Cube<?> cube) {
```

Comment:
@bartoszmajsak The problem of only having one constructor (`@Inject`) is that then I am sure that there will be users that will add two constructors and will complain. With default constructor + setters seems a better approach for avoiding confusions. 

I agree with you that field injection should be supported too.